cranelift/Cargo.toml

```diff
@@ -20,7 +20,7 @@ harness = false
 
 [dependencies]
 cfg-if = { workspace = true }
-cranelift-codegen = { workspace = true, features = ["disas"] }
+cranelift-codegen = { workspace = true, features = ["disas", "trace-log"] }
```

Comment:
I objected to this change during pair programming but Nick convinced me that since the cranelift tools are not performance critical it's actually more useful to leave this always-on in that specific case.

Reply:
This was intentional because `clif-util` is a debugging tool, so it should enable our debugging-related Cargo features. It isn't changing anything about the default features for `cranelift-codegen` itself.

Reply:
Ah, this is for the CLI crate only; I see. Yeah, I think that's actually a useful change.